app/scripts/modules/core/projects/dashboard/cluster/projectCluster.directive.html

```diff
@@ -45,9 +45,12 @@
           </a>
         </td>
         <td>
-          <a class="heavy" ng-if="application.build.buildNumber" href="{{application.build.url}}" target="_blank">
+          <a class="heavy" ng-if="application.build.buildNumber && !application.build.images" href="{{application.build.url}}" target="_blank">
             <span>#</span>{{application.build.buildNumber}}
           </a>
+          <ul class="list-unstyled" ng-if="application.build.images" ng-repeat="image in application.build.images">
```

Comment:
One more thing -- shouldn't the ng-repeat be on the `<li>` insead of the `<ul>`?